kubeless-openshift.jsonnet

```diff
@@ -0,0 +1,10 @@
+# Builds on kubeless-rbac.ksonnet to produce a deployable manifest on OpenShift 1.5
+# Modifies apiVersion for kubeless-controller Deployment to extensions/v1beta1
+# Modifies ClusterRole and ClusterRoleBinding apiVersions to v1
+local kubeless = import "kubeless-rbac.jsonnet";
+
+kubeless + {
+  controller: kubeless.controller + { apiVersion: "extensions/v1beta1" },
```

Comment:
cc @anguslees @ngtuna 

hopefully I'm not doing anything horribly wrong here, let me know if there's a better way to do this!